### PR TITLE
Include elemental commands instead of the cos-* variants

### PR DESCRIPTION
**Pull request by davidcassany**
Signed-off-by: David Cassany <dcassany@suse.com>

### CODE REVIEW

**Comment by davidcassany** [comment]
Related to rancher-sandbox/cOS-toolkit#1144

File: content/en/docs/Getting started/upgrading.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ If running a kubernetes cluster on the `cOS` system, you can leverage the [syste
 apiVersion: upgrade.cattle.io/v1
 kind: Plan
 metadata:
-  name: cos-upgrade
+  name: elemetal-upgrade
```
Comment (Itxaka) [inline content/en/docs/Getting started/upgrading.md:36]: Typo

**Review by Itxaka — CHANGES_REQUESTED** [review]

File: content/en/docs/Getting started/upgrading.md (hunk 1 above)
Comment (davidcassany) [reply content/en/docs/Getting started/upgrading.md:36]: ouch...

**Comment by mudler** [comment]
Maybe since `cosign` changed to be disabled by default, we should mention it in the cosign docs section? https://github.com/rancher-sandbox/cos-toolkit-docs/blob/master/content/en/docs/Creating%20derivatives/cosign.md

**Comment by davidcassany** [comment]
> EDIT: maybe that's not the best place, but it is worth to mention how to enable it during upgrades/install etc.

Yes, that's why I did not include it here. This PR is not adding any new docs just adapting to the current commands. I am working in another PR to add few little things, I can do it there.

**Comment by davidcassany** [comment]
@mudler at the end I did not create another PR, it wasn't that much.

**Review by Itxaka — COMMENTED** [review]
nice!

**Review by Itxaka — APPROVED** [review]

**Review by mudler — APPROVED** [review]

**Pull request merged**